src/backend/user_files.py

```diff
@@ -192,14 +193,35 @@ async def add(self, user_id: str) -> None:
                 "token":      client.access_token,
                 "device_id":  client.device_id,
                 "enabled":    True,
-                "order":      max([
+                "presence":   account.presence.value,
+
+                # Can account.order converge with any other saved value?
+                "order":      account.order if account.order >= 0 else max([
```

Comment:
What did you mean by "converge"?
Also note, negative order are valid values for users to set in their config file, you may want to check for `None` instead

Reply:
By "converge" I mean, if the account with the highest order is manually deactivated, a one added after may take the same order value, because the `account.order` on login is calculated based on active accounts in Mirage at login time.

Reply:
About the negative value, I'm supposing `-1` is the default value and thus it must be calculated a new value for it. But why check it to `None`?

Reply:
> because the account.order on login is calculated based on active accounts

It should consider all the saved accounts from the config file, to avoid clashing order numbers.

> About the negative value, I'm supposing -1 is the default value and thus it must be calculated a new value for it. But why check it to None?

There's no single default value (forget about my `None` comment), every account on login should either use the number from the config file, or assign itself a unique one; then it just use `"order": account.order` when saving here.